fontir/src/ir.rs

```diff
@@ -49,6 +50,7 @@ pub struct Axis {
     pub default: UserSpaceCoord,
     pub max: UserSpaceCoord,
     pub hidden: bool,
+    pub design_to_user: PiecewiseLinearMap,
```

Comment:
designspaceLib calls this just `map`, and -- again -- it's a map from user to design.

each AxisDescriptor has a map_forward and map_backward methods that do the piecewise lerp, where "forward" means from user to design, and "backward" the other way around:

https://github.com/fonttools/fonttools/blob/6aca5be9b772bfb502fcaf29c92a85e3c918ac34/Lib/fontTools/designspaceLib/__init__.py#L929-L945

Reply:
I believe I specifically want design to user because for IR I want user or normalized and *not* design. That probably means I populating it backwards which I will fix.

Reply:
Fixed, provided I'm right that IR wants user and normalized, not design, coords.